.travis.yml

```diff
@@ -14,8 +14,7 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-mono:
-  - 4.0.5
+mono: none
```

Comment:
These changes are a no-op. Just less setup on Travis. EF needs its own strategy for additional cross-platform testing. (Xamarin, UWP, etc.) We can consider adding our dependencies' tests to our infrastructure.

SignalR might want in too if they have a .NET Standard client.